src/main/resources/openapi/courthouse.yaml

```diff
@@ -114,6 +317,38 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/Courthouse'
+
+  responses:
+    ConstraintViolationError:
+      description: Constraint violation
+      content:
+        application/json+problem:
+          schema:
+            $ref: './problem.yaml'
+          example:
+            type: https://zalando.github.io/problem/constraint-violation
```

Comment:
Just to reiterate would we expect specific error codes types relating to each of the validation errors?

Reply:
Although looks like we can influence the response by using ConstraintViolationAdviceTrait